### ISSUE TITLE
Mark styles everywhere

### ISSUE THREAD

**Issue by Made-of-Clay** [SublimeLinter/SublimeLinter#1557]
**Sublime version** 3.1.1 (build 3176)
**SublimeLinter version** 4.9.4
**Vue Syntax Highlight version** 3.3.4
**ESLint Version** 5.13.0

Within the last week, Sublime has been throwing the red squiggly underline mark style all over my Vue SFC. [This screenshot](https://i.imgur.com/yK9dxnn.png) shows the issue (the folded JS has no issues, so isn't shown). I know there isn't anything wrong with what's underlined. It also seems inconsistent when it shows. I haven't nailed down steps to reproduce the marks reliably. 

When I run the file through ESLint via command line, the underlined portions do not show.

This was not an issue as recently as last Friday afternoon.

An example of the console output:
```
SublimeLinter: sublime_linter.py:252  Delay linting 'k-tree-node.vue' for 1.4s
SublimeLinter: sublime_linter.py:252  Delay linting 'k-tree-node.vue' for 1.4s
SublimeLinter: sublime_linter.py:252  Delay linting 'k-tree-node.vue' for 0.0s
SublimeLinter: linter.py:931          eslint: checking lint mode 'background' vs lint reason 'on_save'
SublimeLinter: #1603 linter.py:948    eslint: linting 'k-tree-node.vue'
SublimeLinter: #1603 linter.py:1363   Running ...

  /home/adam/mnt/kona/akona  (working dir)
  $ cat app/modules/job/src/js/k-tree-node.vue | /home/adam/mnt/kona/akona/node_modules/.bin/eslint --format json --stdin --stdin-filename /home/adam/mnt/kona/akona/app/modules/job/src/js/k-tree-node.vue

SublimeLinter: #1603 linter.py:1027   eslint: output:
  [{"filePath":"/home/adam/mnt/kona/akona/app/modules/job/src/js/k-tree-node.vue","messages":[],"errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0}]
SublimeLinter: #1603 linter.py:73     eslint output:
[{'errorCount': 0,
  'filePath': '/home/adam/mnt/kona/akona/app/modules/job/src/js/k-tree-node.vue',
  'fixableErrorCount': 0,
  'fixableWarningCount': 0,
  'messages': [],
  'warningCount': 0}]
SublimeLinter: sublime_linter.py:449  Linting 'k-tree-node.vue' took 2.99s

```

**Comment by braver** [comment]
Do you also have some debug output from ice SublimeLinter?

**Comment by kaste** [comment]
Hi @Made-of-Clay!

Wow, that's a red experience. How do you know that these squiggles actually come from SublimeLinter? I would expect some outline marks here as well. Do these errors also appear in SublimeLinter's error panel? The console output does report the file as clean. Do you also have the squiggles for that particular file?

**Comment by Made-of-Clay** [comment]
@braver I'm not sure what you mean by "ice SublimeLinter" ... sorry :-/

@kaste In fairness, I don't know it's SublimeLinter. That was just my best guess. When eslint notifies me of stuff, it outlines it (yellow or red). I guessed SublimeLinter because it had a term for the Google-searching I was doing (e.g. "sublime lint red squiggly underline" which led me to 3.10.10's [Mark Styles](http://www.sublimelinter.com/en/v3.10.10/mark_styles.html) page; an older version yes, but that's the closest I got -- I'm also noting that my search kind of targeted SublimeLinter with the terms "sublime" and "lint"... hm...). I've got SublimeLinter's `debug` setting as `true`, but don't know where else to look for output beyond what I copied originally (sublime's console). I *do* (sometimes) get the hover tip error. For example, hovering the first `<div` shows...
```
Cannot find name 'div'.
any
```
... and hovering that div's `class` attribute shows...
```
'>' expected.
(local class) (Anonymous class)
```
If I'm missing where to find another error console, let me know where and I'll report back.

**Comment by kaste** [comment]
Ah well, maybe you configured SublimeLinter globally to lint Vue files but you actually have the Vue plugin for eslint only installed locally per project.

**Comment by Made-of-Clay** [comment]
Hmm... is it plausible that something would have changed in the last week that would cause such a configuration to give me my current problems? I never considered installing `eslint-plugin-vue` globally.

**Comment by braver** [comment]
Sorry, typo!

Please check this guide: 
http://www.sublimelinter.com/en/stable/troubleshooting.htm

Let us know what’s going on under the hood by sharing your SublimeLinter settings and a debug log, otherwise we’re just guessing.

**Comment by Made-of-Clay** [comment]
@braver That appears to be a broken link (missing "l" on ".html"). I found [the page](http://www.sublimelinter.com/en/stable/troubleshooting.html) you were trying to reference (I've been on this page while looking for answers before posting this Issue). As I mentioned in my first response to @kaste, I have the `debug` flag/option set to `true`, and the resulting output is what I referenced in my 1st post. To confirm this was the case, I set `debug` to `false`, and nothing shows in the console when the linter runs.

In short, the log output you're looking for is already in the original post that opened this Issue.

**Comment by Made-of-Clay** [comment]
I'm going to close this... coming into work this morning and looking at the file again, the hover tooltip things aren't showing anymore (but the marks are still showing). I closed and re-opened the file and the marks haven't shown again yet. As I mentioned in the 1st post, it's inconsistent.

I don't feel I have enough info to provide to [help anyone help me](https://gph.is/2ypL3Sd)... If I ever learn what's going on, I will be sure to post back for future reference. Thanks @braver and @kaste for your efforts!
